db/migrate/20190318094437_add_imported_attr_to_ansible_variables.rb

```diff
@@ -0,0 +1,15 @@
+class AddImportedAttrToAnsibleVariables < ActiveRecord::Migration[5.2]
+  def up
+    add_column :lookup_keys, :imported, :boolean
+
+    AnsibleVariable.find_in_batches do |batch|
+      batch.map do |variable|
+        variable.update_attribute :imported, true
```

Comment:
Migrations should take care of updating the existing variables, it works for me without problems.